### PR TITLE
cmd: fix link to TeamCity build log

### PR DESCRIPTION
**Pull request by jordanlewis**
Link to the build log directly instead of the the test overview which is
usually not as useful.

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/10905)

### CODE REVIEW

File: pkg/cmd/github-post/main.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ const pkgEnv = "PKG"
 // 422 Validation Failed [{Resource:Issue Field:body Code:custom Message:body is too long (maximum is 65536 characters)}]
 //
 // Subtract some length just to be safe.
-const githubIssueBodyMaximumLength = 1<<16 - 1<<10
+const githubIssueBodyMaximumLength = 1<<16 - 1<<11
```
Comment (tamird) [inline pkg/cmd/github-post/main.go]: This was actually necessary? Urgh, perhaps it's time we did this calculation for real.

**Review by tamird — APPROVED** [review]
Ah, indeed, the build log is useless because we don't use go-test-teamcity here. LGTM.

File: pkg/cmd/github-post/main.go (hunk 1 above)
Comment (jordanlewis) [inline pkg/cmd/github-post/main.go]: Yeah necessary. The math also doesn't work. I'm fixing this to do the calculation for real.

**Comment by jordanlewis** [comment]
Review status: 1 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[pkg/cmd/github-post/main.go, line 47 at r1](https://reviewable.io:443/reviews/cockroachdb/cockroach/10905#-KX6pjWlzckPNicDjfDu-r1-47:-KX771K1LIaNabEGoLsc:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/01ad8cc1987c84a9225c625ff390fd04478f1e59/pkg/cmd/github-post/main.go#L47)):*
<details><summary><i>Previously, tamird (Tamir Duberstein) wrote…</i></summary>
> Mind making this `1 << 16 - 1` just to be safe?
</details>
Done.

---

*[pkg/cmd/github-post/main.go, line 115 at r2](https://reviewable.io:443/reviews/cockroachdb/cockroach/10905#-KX75LffYNwgzred9-x-:-KX770zfzVsosC2KHOBG:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/46ed60993585b12ca383ce810acd4fa96d3958a5/pkg/cmd/github-post/main.go#L115)):*
<details><summary><i>Previously, tamird (Tamir Duberstein) wrote…</i></summary>
> this is unusual, so a comment wouldn't be out of place.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/10905)*

**Comment by tamird** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks pending.

---

*[pkg/cmd/github-post/main.go, line 116 at r3](https://reviewable.io:443/reviews/cockroachdb/cockroach/10905#-KX78UsMKxoaXoCIFNf4:-KX78UsMKxoaXoCIFNf5:bedpadp) ([raw file](https://github.com/cockroachdb/cockroach/blob/0df63b9383a6e774a259e2c609a634e1d58753ad/pkg/cmd/github-post/main.go#L116)):*
> ```Go
> 
> %s`, sha, u.String(), "```\n%s\n```")
> 		// We insert a raw "%s" above so we can figure out the length of the body so far, without
> ```

I think this is wrapped incorrectly, but I can fix it in my PR that's about to follow this one if you prefer to just merge this.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/10905)*

**Comment by tamird** [comment]
Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/10905)*

**Pull request merged**